beetsplug/vocadb.py

```diff
@@ -181,24 +195,33 @@ def __init__(self) -> None:
         self.config.add(
             {
                 "source_weight": 0.5,
-                "prefer_romaji": False,
-                "translated_lyrics": False,
-                "include_featured_album_artists": False,
-                "va_string": DEFAULT_VA_STRING,
             }
         )
+        self.config.add(self.default_config)
+
+        self.data_source: str = self.instance_info.name
 
     def __init_subclass__(cls, instance_info: InstanceInfo) -> None:
         super().__init_subclass__()
         cls.instance_info = instance_info
+        vocadb_config = config["vocadb"]
+        if vocadb_config.exists():
+            for key in vocadb_config.keys():
+                cls.default_config[key] = vocadb_config[key].get()
```

Comment:
If I'm reading this right, this is only used by `get_artists` function to get the current `va_string` config option?

If this is the case I think this can be dropped, see my comment on that function.

Reply:
Nevermind, I just read the README changes closer and I now see what this is actually for. But still, see my comment on `get_artists` function.